recipe/meta.yaml

```
@@ -56,6 +60,31 @@ outputs:
         convenient.
 
         This package contains utility modules to talk to Avahi with Python easier.
+  - name: avahi
+    script: lib-avahi.sh
+    requirements:
+      build:
+        # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+        # Compilers are named 'c', 'cxx' and 'fortran'.
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - make
+        - pkg-config
+        - intltool
+        - make
+      host:
+        - libevent
+        - libdaemon
+        - dbus >=0.34
+    test:
+      requires:
+        - dbus >=0.34
+        - libdaemon
+        - libevent
```

I think there's a few build requirements missing (based on testing with `conda build --error-overlinking --error-overdepending`, and the `test/requires` here should only include the optional dependencies required for the tests we want to run - it looks like the list here is just the runtime requirements, which should be in `requirements/run`.

```suggestion
      host:
        - expat
        - glib
        - libevent
        - libdaemon
        - dbus >=0.34
      run:
        - dbus >=0.34
        - expat
        - libdaemon
        - libevent
        - libglib
    test:
```